plugins/com.google.cloud.tools.eclipse.appengine.newproject.test/src/com/google/cloud/tools/eclipse/appengine/newproject/AppEngineStandardWizardPageTest.java

```diff
@@ -0,0 +1,15 @@
+package com.google.cloud.tools.eclipse.appengine.newproject;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class AppEngineStandardWizardPageTest {
```

Comment:
Hard to test too much without calling createControl. The error message should initially be null, I thought?